Sources/SWBUniversalPlatform/Specs/Ld.xcspec

```diff
@@ -400,9 +400,9 @@
                 CommandLineArgs = {
                     "" = ();
                     "<<otherwise>>" = (
-                        "-Xlinker",
-                        "-dylib_install_name",
-                        "-Xlinker",
+                        "-Xclang-linker",
+                        "-install_name",
```

Comment:
-install_name translates to -dylib_install_name on Mach-O platforms and -soname on ELF platforms?

Reply:
Sadly, `-install_name` is a driver flag that is Darwin specific, and will be translated to `-Xlinker -dylib_install_name`. It has no impact on ELFish targets.